core/templates/components/forms/custom-forms-directives/thumbnail-display.component.ts

```diff
@@ -0,0 +1,65 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Component for thumbnail display.
+ */
+
+import { Component, Input, OnChanges, OnInit } from '@angular/core';
+import { downgradeComponent } from '@angular/upgrade/static';
+
+import { SvgSanitizerService } from 'services/svg-sanitizer.service';
+
+@Component({
+  selector: 'oppia-svg-thumbnail-display',
```

Comment:
are we expecting this to be `oppia-thumbnail-display`?

Reply:
Yes, we are. Thanks for pointing out.